roles/edge-image-builder/templates/rke2-config-mgmt.yaml.j2

```diff
@@ -0,0 +1,8 @@
+cni: {{cluster_cni}}
+{% if libvirt_network_ipv6|bool %}
```

Comment:
Maybe we can already put in place support for single v4, single v6 and dual here.

Reply:
I have added initial support for single-stack v6, not yet tested though and we know e.g Rancher doesn't yet support it so the management cluster at least probably has to remain dual-stack